bindings/rust/s2n-tls/src/testing/resumption.rs

```diff
@@ -63,13 +64,15 @@ mod tests {
     }
 
     #[test]
-    fn resume_session() -> Result<(), Box<dyn Error>> {
+    fn resume_tls12_session() -> Result<(), Box<dyn Error>> {
         let keypair = CertKeyPair::default();
 
+        let tls12_policy = Policy::from_version("20240501")?;
```

Comment:
Our Rust tests are generally higher level than our C unit tests. Rather than this opaque version with no explanation, maybe you could create a crate-visible named policy like DEFAULT_TLS12 or TEST_TLS12 for testing.